extensions/scheduler/deployment/src/test/java/io/quarkus/scheduler/test/DisabledScheduledMethodTest.java

```diff
@@ -0,0 +1,49 @@
+package io.quarkus.scheduler.test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.scheduler.Scheduled;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class DisabledScheduledMethodTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest test = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addClasses(Jobs.class)
+                    .addAsResource(new StringAsset("DisabledScheduledMethodTest.interval=off"),
+                            "application.properties"));
+
+    @Test
+    public void testNoSchedulerInvocations() throws InterruptedException {
+        Thread.sleep(100);
+        Jobs.LATCH.await(500, TimeUnit.MILLISECONDS);
+        assertEquals(0, Jobs.executionCounter);
+    }
+
+    static class Jobs {
+
+        static final CountDownLatch LATCH = new CountDownLatch(1);
+
+        static volatile int executionCounter = 0;
+
+        @Scheduled(every = "{DisabledScheduledMethodTest.interval}")
```

Comment:
I think it would be good to test a method that is disabled with a default value too, i.e. just add something like:
```java
@Scheduled(every = "{non.existent.property:disabled}")
void disabledByDefault() {
   executionCounter++;
}
```

Reply:
Yes, sure